Assets/Scripts/SS3D/Systems/Inventory/Items/ItemActor.cs

```diff
@@ -91,20 +82,24 @@ public Sprite InventorySprite
         /// </summary>
         public Container Container
         {
-            get => _container;
+            get => item.Container;
             set => SetContainer(value, false, false);
         }
 
-        public void Awake()
+        public new void Awake()
         {
-            _sprite = null;
+            item.Name = _startingName;
+            item.Weight = _startingWeight;
+            item.Traits = _startingTraits;
+
+            item.Sprite = null;
 
```

Comment:
Empty the starting traits list, set starting name string to "", free as much memory as possible here.